src/components/ProfilePage/AddressMenu/AddressMenu.tsx

```diff
@@ -0,0 +1,84 @@
+import ClickAwayListener from '@mui/material/ClickAwayListener';
+import Fade from '@mui/material/Fade';
+import { MenuPaper, MenuPopper } from 'src/components/Menu/Menu.style';
+import { MenuItem } from 'src/components/Menu/MenuItem';
+import { useAddressMenuContent } from '../useAddressMenuContent';
+import { AddressMenuList } from './AdressMenu.style';
+
+export const AddressMenu = ({
+  open,
+  setOpen,
+  anchorEl,
+}: {
+  open: boolean;
+  setOpen: any;
```

Comment:
Can you type this to avoid issues later on

Reply:
yes